### PR TITLE
Add flatpak files

### PR DESCRIPTION
**Pull request by kutzlect**
Here's the long awaited PR with the flatpak files. I will delete the other repo upon merge as I would like to continue the maintenance here.

### CODE REVIEW

File: flatpak/io.github.thaunknown.miru.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <url type="help">https://discord.com/invite/Z87Nh7c4Ac</url>
+    <url type="donation">https://github.com/sponsors/ThaUnknown</url>
+    <description>
+        <p>
+            Bittorrent streaming software for cats. Stream anime torrents, real-time with no waiting for downloads.
+        </p>
+    </description>
+        <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/kutzlect/io.github.thaunknown.miru/master/miru-screenshot-220830.jpg</image>
```
Comment (ThaUnknown) [inline flatpak/io.github.thaunknown.miru.metainfo.xml:27]: ?

**Comment by ThaUnknown** [comment]
@Muril-o could ya review this?

**Review by Muril-o — COMMENTED** [review]
Is there a reason for the flatpak files to be on this repository?
Since the install button leads to the [flathub repository](https://github.com/flathub/io.github.thaunknown.miru), can't maintenance be done solely on the flathub repository? For me it feels unnecessary to maintain 4 separate files across 2 different repos, when you could maintain only the flathub repo.

**Comment by Vysp3r** [comment]
@Muril-o You're absolutely right, there's no need for you to maintain Flatpak files on your side.

**Comment by orowith2os** [comment]
The metadata file here isn't Flatpak-specific, and should be maintained here.

**Pull request closed without merging**